.gitignore

```diff
@@ -2,88 +2,148 @@
 PATHS-common.txt
 PATHS-detection.txt
```

Comment:
These first three lines may be removed. They're hold-overs from a specific branch of the mono-repo.